asm/nop.s

```diff
@@ -0,0 +1,4 @@
+  .global __nop
+__nop:
+  nop
```

Comment:
We could drop the `nop` instruction here. Even without it `asm::nop` won't be optimized away by the compiler.